systems/plants/RigidBody.cpp

```diff
@@ -8,6 +8,7 @@ const int defaultRobotNum[1] = {0};
 const set<int> RigidBody::defaultRobotNumSet(defaultRobotNum,defaultRobotNum+1);
 
 RigidBody::RigidBody(void) :
+    parent(nullptr),
```

Comment:
nullptr is fine and we should use it everywhere. We support a subset of c++11 features (whatever is supported by MSVC2010, the lowest common denominator). If the makefiles don't reflect that, we should change them.